Configuration/TCA/tx_html5mediakit_domain_model_media.php

```diff
@@ -86,6 +86,15 @@
                 'maxitems' => 1,
             ],
         ],
+        'subtitles' => [
```

Comment:
To be consistent with the HTML element, it would be nice if the field was named `tracks` instead of `subtitles`.